src/errors.ts

```diff
@@ -80,21 +80,28 @@ export function throwSyntaxError(message: string, source: SrcInfo): never {
     );
 }
 
-export function throwCompilationError(message: string, source: SrcInfo): never {
-    throw new TactCompilationError(
-        `${locationStr(source)}${message}\n${source.interval.getLineAndColumnMessage()}`,
-        source,
-    );
+export function throwCompilationError(
+    message: string,
+    source?: SrcInfo,
+): never {
+    const loc =
```

Comment:
It's now less informative in some cases. For example, when trying to compile non-existing project source:
```
💼 Compiling project getters ...
Tact compilation failed
Error
    at throwCompilationError (/Users/daniil/Coding/tact/src/errors.ts:91:11)
    at resolveImports (/Users/daniil/Coding/tact/src/imports/resolveImports.ts:28:30)
    at precompile (/Users/daniil/Coding/tact/src/pipeline/precompile.ts:18:36)
    at build (/Users/daniil/Coding/tact/src/pipeline/build.ts:73:25)
    at run (/Users/daniil/Coding/tact/src/node.ts:130:34)
    at async /Users/daniil/Coding/tact/scripts/prepare.ts:20:31
Error: Tact projects compilation failed
    at /Users/daniil/Coding/tact/scripts/prepare.ts:24:19
error Command failed with exit code 1.
```

while before it would throw `Could not find entrypoint XXX`.

I think the problem is here.